src/page-modules/assistant/details/trip-section/estimated-calls-section.tsx

```diff
@@ -2,30 +2,53 @@ import { Typo } from '@atb/components/typography';
 import { TripRow } from '@atb/modules/trip-details';
 import { PageText, useTranslation } from '@atb/translations';
 import { secondsToDuration } from '@atb/utils/date';
+import Link from 'next/link';
 
 import style from './trip-section.module.css';
 
 export type EstimatedCallsSectionProps = {
   numberOfIntermediateEstimatedCalls: number;
   duration: number;
+  serviceJourneyId: string | null;
+  date: string;
+  fromQuayId: string | null;
 };
 
 export function EstimatedCallsSection({
   numberOfIntermediateEstimatedCalls,
   duration,
+  serviceJourneyId,
+  date,
+  fromQuayId,
 }: EstimatedCallsSectionProps) {
   const { t, language } = useTranslation();
 
+  const shouldShowLinkToServiceJourney = serviceJourneyId ? true : false;
```

Comment:
This can be simplified, but maybe it's not cognitively simpler.

```suggestion
  const shouldShowLinkToServiceJourney = !!serviceJourneyId;
```

Reply:
I think this is better 👍